### PR TITLE
Make header and content optional

### PR DESCRIPTION
**Pull request by SiAdcock**
To support things like this:

picture 9

And this:

picture 10

### CODE REVIEW

File: src/js/text/carousel.dot.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <ol class="brexit__carousel-content">
         {{~it.data.slides :value:index}}
         <li class="brexit__carousel-slide{{? index===0}} brexit__carousel-slide--current js-current{{?}}" id="carousel-slide-{{=index}}">
-            {{? value.headline}}
+            {{? typeof value.headline !== 'undefined'}}
```
Comment (joelochlann) [inline src/js/text/carousel.dot.html]: Haven't we covered this case in the code in `textCarousel.js` above?

**Comment by joelochlann** [comment]
👍  apart from one minor question

**Comment by joelochlann** [comment]
actually please hold off merging until #19 is merged - I keep getting new conflicts whenever anything else goes into master!

**Comment by SiAdcock** [comment]
Since the template is shared between big number and text carousels, this check is to ensure the `brexit__header` is only included if there is a `headline` property. It's essentially a proxy for a check to see which type of carousel we are building.

I'm open to doing it another way; perhaps we can add a `format` property to the `data` object which could take the value `text_carousel` or `number_carousel`?

**Comment by joelochlann** [comment]
Ah yeah, forgot the template was shared. All looks good! 👍 🐺  🐶

**Pull request merged**